test/calculatorW2Gtest.cpp

```diff
@@ -0,0 +1,99 @@
+/**
+ * @file calculatorW2Gtest.cpp
+ * @author Gary Dean Jenkins (gary.jenkins@intimetec.com)
+ * @brief 
+ * @version 0.1
+ * @date 2021-07-20
+ * 
+ * @copyright Copyright (c) 2021
+ * 
+ */
+
+#include <gtest/gtest.h>
+#include <gmock/gmock.h>
+#include "../inc/CalculatorW2.h"
+#include "../inc/CalculatorStrings.h"
+
+using namespace calculatorw2;
+
+TEST(WeekTwoCalculatorW2Tests, TheRequiredMathAndControlOperatorsMustBeSupported)
+{
+    EXPECT_EQ((char)Operators::UNKNOWN, 'u');
+    EXPECT_EQ((char)Operators::NOOP, 'n');
+    EXPECT_EQ((char)Operators::HELP, 'h');
+    EXPECT_EQ((char)Operators::QUIT, 'q');
+    EXPECT_EQ((char)Operators::ADD, '+');
+    EXPECT_EQ((char)Operators::SUBTRACT, '-');
+    EXPECT_EQ((char)Operators::MULTIPLY, '*');
+    EXPECT_EQ((char)Operators::DIVIDE, '/');
+}
+
+
+TEST(WeekTwoCalculatorW2Tests, WhenACalculatorW2IsCreatedThenTheErrorMustBeNull)
```

Comment:
Look into Test Fixtures, instantiation of CalculatorW2 could be moved into a fixture since it is common for most of these tests